jms/src/test/java/akka/stream/alpakka/jms/javadsl/JmsConnectorsTest.java

```diff
@@ -332,6 +334,9 @@ public void publishAndConsumeJmsTextMessagesWithHeaders() throws Exception {
                     .map(jmsTextMessage -> jmsTextMessage.withHeader(JmsType.create("type")))
                     .map(jmsTextMessage -> jmsTextMessage.withHeader(JmsCorrelationId.create("correlationId")))
                     .map(jmsTextMessage -> jmsTextMessage.withHeader(JmsReplyTo.queue("test-reply")))
+                    .map(jmsTextMessage -> jmsTextMessage.withHeader(JmsTimeToLive.create(999, TimeUnit.SECONDS)))
```

Comment:
Not really changed in this PR, but wouldn't it be much better to chain all `withHeader` calls in one map operation?

Reply:
Of course. Fixed it and added documentation of sending messages with headers.